src/plugin/vault/vault_test.go

```diff
@@ -313,20 +451,20 @@ func TestRenewAfterTTLExpires(t *testing.T) {
 }
 
 func TestRevoke(t *testing.T) {
-	ttl := 2
-	c, v, secret := loginHelper(t, fmt.Sprintf("%vs", ttl))
+	c, v, secret := loginHelper(t, "")
+	if secret.LeaseDuration < 60 {
```

Comment:
Where does 60 come from? If that's a default when no ttl param or config is specified, where is it defined? We should use that const in the tests if possible.

Reply:
The maximum lease duration is determined by vault. I think it's actually 30 days, but I've only seen that value in their documentation.